### PR TITLE
'Open/Close Floor Flusher' Door Timer Buttons

### PR DESCRIPTION
**Pull request by Mister-Moriarty**
[QoL] [Feature] [tgui]

## About the PR:
Adds a new section to the brig timer tgui: "Floor Flusher", which allows control over whether the floorflusher is open or closed. See screenshots below:

Eg 1  Eg 2
Eg 3  Eg 4

## Why's this needed?
On occasion, someone will refuse to leave the brig, and it would be useful were there an option to close the flusher, so something doesn't have to be thrown down it. On other occasions, it is useful to have the ability immediately release someone from the brig, or if a group of people need released.

## Changelog:
```changelog
(u)Mr. Moriarty
(+)Added 'Open/Close Floor Flusher' brig door timer buttons.
```

### CODE REVIEW

File: code/obj/machinery/door/door_timer.dm [hunk 1]
```diff
@@ -262,6 +262,15 @@
 			)
 			break
 
+	for (var/obj/machinery/floorflusher/FF in range(30, src))
```
Comment (ZeWaka) [inline code/obj/machinery/door/door_timer.dm:265]: 30????????

File: code/obj/machinery/door/door_timer.dm [hunk 2]
```diff
@@ -310,6 +319,20 @@
 					logTheThing("station", usr, null, "sets off flashers from a door timer: [src] [log_loc(src)].")
 					return TRUE
 
+		if ("toggle-flusher")
+			for (var/obj/machinery/floorflusher/FF in range(30, src))
```
Comment (ZeWaka) [inline code/obj/machinery/door/door_timer.dm:323]: 30????????

Comment (Mister-Moriarty) [reply code/obj/machinery/door/door_timer.dm:323]: That is the range check used for the floorflusher on line 221, which is clarified on line 192, saying that "Cog2 places linked fixtures much further away from the timer than originally envisioned."
I am unsure if this still holds true, however.

Comment (ZeWaka) [reply code/obj/machinery/door/door_timer.dm:323]: ok can be fixed later, as this will lead to same perf

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**